src/webrtc/callEventHandler.ts

```diff
@@ -255,10 +257,14 @@ export class CallEventHandler {
                     this.calls.delete(content.call_id);
                 }
             }
+            return;
         }
 
-        // The following events need a call
-        if (!call) return;
+        // The following events need a call and a peer connection
+        if (!call && !call.hasPeerConnection) {
```

Comment:
```suggestion
        if (!call || !call.hasPeerConnection) {
```

Sounds saner to me, am I missing something?

Reply:
Oh yeah, right.